### PR TITLE
add `AcquisitionDuration`

### PR DESCRIPTION
**Pull request by alexfoias**
Units of `AcquisitionDuration` in seconds based on https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/01-magnetic-resonance-imaging-data.html

Fixes https://github.com/sct-pipeline/gm-challenge-data/issues/25

### CODE REVIEW

**Comment by alexfoias** [comment]
@jcohenadad I added the duration based on the `sub-XXX_acq_params.pdf` file.

I have to recheck the changes again before giving the final okay.

File: sub-9709Ses2/anat/sub-9709Ses2_run-1_T2starw.json [hunk 1]
```diff
@@ -3,5 +3,6 @@
 	"MagneticFieldStrength": 1.5,
 	"Manufacturer": "Siemens",
 	"ManufacturersModelName": "Avanto",
-	"InstitutionName": "Milan"
+	"InstitutionName": "Milan",
+	"AcquisitionDuration": "482"
```
Comment (jcohenadad) [inline sub-9709Ses2/anat/sub-9709Ses2_run-1_T2starw.json:7]:
```suggestion
	"AcquisitionDuration": "462"
```

File: sub-9709Ses2/anat/sub-9709Ses2_run-2_T2starw.json [hunk 2]
```diff
@@ -3,5 +3,6 @@
 	"MagneticFieldStrength": 1.5,
 	"Manufacturer": "Siemens",
 	"ManufacturersModelName": "Avanto",
-	"InstitutionName": "Milan"
+	"InstitutionName": "Milan",
+	"AcquisitionDuration": "482"
```
Comment (jcohenadad) [inline sub-9709Ses2/anat/sub-9709Ses2_run-2_T2starw.json:7]:
```suggestion
	"AcquisitionDuration": "462"
```

**Review by jcohenadad — APPROVED** [review]
thanks!

**Pull request merged**